utilities/four_c_utils/src/four_c_utils/check_workflow_dependencies_hash.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of 4C multiphysics licensed under the
+# GNU Lesser General Public License v3.0 or later.
+#
+# See the LICENSE.md file in the top-level for license information.
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+import subprocess
+import os
+import yaml
+import sys
+
+
+def compute_dependencies_hash():
+    result = subprocess.run(
+        r"find dependencies docker -not -wholename '*/trilinos_develop/*' -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8",
```

Comment:
We are duplicating this line in a few places already. Can we extract it somewhere into a script?

Reply:
True. I'll do it once we have synchronized with Gitlab.